frontends/web/src/locales/en/app.json

```diff
@@ -1059,6 +1062,10 @@
       }
     },
     "advancedSettings": {
+      "authentication": {
+        "description": "Requires authentication to use the app.",
+        "title": "Enable authentication"
```

Comment:
phoenix calls this "Screen lock" and description "Lock access to the app with Android's screen lock/fingerprint". I think these fit better, as "authentication" is not very clear imo.